src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="display">
+        <input type="text" value="0" readOnly />
+      </div>
+      <div className="buttons">
+        <div className="row">
+          <button type="button" className="number">AC</button>
+          <button type="button" className="number">+/-</button>
+          <button type="button" className="number">%</button>
+          <button type="button" className="operator">÷</button>
+        </div>
```

Comment:
- As stated in the project requirement, you need to follow the styes, design, and colors, The color of these operators is black according to the provided design, but yours is white, i'll suggest you update them accordingly 

- - Your design:
![Screenshot from 2023-09-12 16-06-33](https://github.com/Yonasle/math_magicians/assets/107632780/299d64cc-74de-416d-84f8-5f0d8c162c36)

- - Provided design:
 
![Screenshot from 2023-09-12 16-09-10](https://github.com/Yonasle/math_magicians/assets/107632780/5728e48a-fe01-4717-aca1-9985d067ee7e)